src/uct/base/uct_iface.c

```diff
@@ -39,7 +39,7 @@ const char *uct_ep_operation_names[] = {
     [UCT_EP_OP_RNDV_ZCOPY]   = "rndv_zcopy",
     [UCT_EP_OP_ATOMIC_POST]  = "atomic_post",
     [UCT_EP_OP_ATOMIC_FETCH] = "atomic_fetch",
-    [UCT_EP_OP_LAST]         = NULL
+    [UCT_EP_OP_LAST]         = "unspecified"
```

Comment:
maybe we should not create a perf_node when op==UCT_EP_OP_LAST? i.e skip calling ucp_proto_perf_node_new in this case, when name is supposedly NULL

Reply:
Done.